src/main/java/org/entur/gbfs/GbfsSubscription.java

```diff
@@ -54,7 +54,8 @@ public void init() {
                 subscriptionOptions.getDiscoveryURI().toString(),
                 subscriptionOptions.getHeaders(),
                 subscriptionOptions.getLanguageCode(),
-                subscriptionOptions.getRequestAuthenticator()
+                subscriptionOptions.getRequestAuthenticator(), //
+                Optional.empty()
```

Comment:
Should it be possible to provide timeout values via the subscription options?

Reply:
Done